src/project.rs

```diff
@@ -63,6 +63,8 @@ pub struct Project {
 
     #[serde(skip_deserializing,skip_serializing)]
     pub template_data: BTreeMap<&'static str, String>,
+
+    pub reuse_private_key: Option<bool>,
```

Comment:
This should use a serde default flag instead of an `Option` since it will always have a value.

Reply:
I struggled with this a bit and got some horrible compiler errors. Could you point me in the right direction?

Reply:
Unfortunately, serde default needs to be a function.

You can use something like [this](https://github.com/Dynulo/Gluon/blob/master/src/files/config.rs#L48), then we can add [this](https://github.com/Dynulo/Gluon/blob/master/src/files/config.rs#L54) to main.rs and use `crate::dft_false` as our default value. It isn't ideal, but it is better than an option.